components/list/list-item-mixin.js

```diff
@@ -682,7 +695,7 @@ export const ListItemMixin = superclass => class extends composeMixins(
 					class="d2l-list-item-actions-container">
 					<slot name="actions" class="d2l-list-item-actions">${actions}</slot>
 				</div>
-				${this._showAddButton ? html`
+				${this._showAddButton && !this._hasNestedListAddButton ? html`
```

Comment:
We don't want the add button for the last item of the list showing if there is a nested list below it.
We would want it showing if the nested list is collapsed though, so I'll do a bit more playing with this.

Reply:
I've dealt with the expand/collapse state piece in the follow up nested PR.